packages/cli/src/commands/create/index.ts

```diff
@@ -131,21 +142,55 @@ async function _createTutorial(flags: CreateOptions) {
 
   updatePackageJson(resolvedDest, tutorialName, flags);
 
-  const { selectedPackageManager, dependenciesInstalled } = await installDependencies(resolvedDest, flags);
+  const selectedPackageManager = await selectPackageManager(flags);
 
   updateReadme(resolvedDest, selectedPackageManager, flags);
 
   await setupEnterpriseConfig(resolvedDest, flags);
 
   await initGitRepo(resolvedDest, flags);
 
+  const { install, start } = await installAndStart(flags);
+
   prompts.log.success(chalk.green('Tutorial successfully created!'));
 
-  printNextSteps(dest, selectedPackageManager, dependenciesInstalled);
+  if (install || start) {
+    let message = 'Please wait while we install the dependencies and start your project...';
 
-  prompts.outro(`You're all set!`);
+    if (install && !start) {
+      // change the message if we're only installing dependencies
+      message = 'Please wait while we install the dependencies...';
 
-  console.log('Until next time 👋');
+      // print the next steps without the install step in case we only install dependencies
+      printNextSteps(dest, selectedPackageManager, true);
```

Comment:
If we install the dependencies, but we don't start the project, we still print the next steps without the `install` step.

```
◆  Tutorial successfully created!
│
│  Next Steps
│
◇  1. cd ./dry-king - Navigate to project
│
◇  2. npm run dev - Start development server
│
◇  3. Head over to http://localhost:4321
│
└  Please wait while we install the dependencies...
```